apps/examples/testcase/ta_tc/audio/utc/utc_audio_main.c

```diff
@@ -738,7 +738,7 @@ static void utc_audio_pcm_writei_p(void)
 	g_pcm = pcm_open(0, 0, PCM_OUT, NULL);
 	TC_ASSERT_GT("pcm_writei", pcm_is_ready(g_pcm), 0);
 
-	size = pcm_get_buffer_size(g_pcm);
+	size = pcm_frames_to_bytes(g_pcm, pcm_get_buffer_size(g_pcm));
 	buffer = malloc(size);
 	TC_ASSERT_NEQ_CLEANUP("pcm_writei", buffer, NULL, clean_all_data(0, NULL));
 
```

Comment:
I think we should change malloc bytes in readi_n and write_n tc also, although the allocated buffer is never used there

Reply:
Yes altough.....
Anyway done.